lib/findpath.mjs

```diff
@@ -0,0 +1,58 @@
+import { dirname } from 'node:path';
+import { env, platform } from 'node:process';
+import { join, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+/**
+ * Get the platform dependant path of the NW.js or ChromeDriver binary.
+ * 
+ * @param {'nwjs' | 'chromedriver'} executable Path to NW.js or Chromedriver executable.
+ * @return {string}
+ */
+export default function findpath(executable = 'nwjs') {
+  const __dirname = dirname(fileURLToPath(import.meta.url));
+  const nwDir = resolve(__dirname, '..', 'nwjs');
+  /**
+   * File path to executable.
+   * 
+   * @type {string}
+   */
+  let binPath = '';
+
+  /**
+    * Host operating system
+    * 
+    * @type {NodeJS.Platform}
+    */
+  let hostOs = env.npm_config_nwjs_platform || env.NWJS_PLATFORM || platform;
+
+  /**
+   * Get the platform dependant path of the NW.js binary.
+   */
+  function findNwjs() {
+    if (hostOs === 'darwin') {
+      binPath = join(nwDir, 'nwjs.app', 'Contents', 'MacOS', 'nwjs');
```

Comment:
In the original code, the logic was like this:
```suggestion
    if (hostOs === 'darwin') {
      if (fs.existsSync(path.join(nwDir, 'Contents'))) {
        binPath = path.join(nwDir, 'Contents', 'MacOS', 'nwjs');
      } else {
        binPath = path.join(nwDir, 'nwjs.app', 'Contents', 'MacOS', 'nwjs');
      }
```

Reply:
I'm not sure why this check was introduced - maybe to cater to a specific requirement for MacOS app bundling? I do know that for v0.81.0 onward, this check is not required since the download is a folder called `nwjs` which contains `nwjs.app` inside it.

Reply:
I think something happened where the location needed to change, so old versions of NW.js used one path, and new versions use a different path. Not sure if this was due to something in Chromium or OSX changing, or both. Since we are only focusing on the latest version, you are correct, this does not need re-implemented.